src/Umbraco.Web/PublishedCache/NuCache/DataSource/PropertyData.cs

```diff
@@ -28,22 +33,26 @@ public string Segment
             set => _segment = value ?? throw new ArgumentNullException(nameof(value)); // TODO: or fallback to string.Empty? CANNOT be null
         }
 
+        [DataMember(Order = 2)]
         [JsonProperty("v")]
         public object Value { get; set; }
 
         //Legacy properties used to deserialize existing nucache db entries
+        [DataMember(Order = 3)]
         [JsonProperty("culture")]
         private string LegacyCulture
```

Comment:
These legacy properties could be removed if a migration forced a rebuild of the nucache table in the database. But they are currently necessary as umbraco will fail to boot otherwise

Reply:
yep we'll need to keep them unfortunately